inception-ui-kb/src/main/java/de/tudarmstadt/ukp/inception/ui/kb/ConceptTreePanel.html

```diff
@@ -37,6 +37,7 @@ <h3 class="panel-title">
     </div>
     <div class="panel-footer text-right">
       <button wicket:id="add" type="submit" class="btn btn-primary"><i class="fa fa-plus" aria-hidden="true"></i> <wicket:container wicket:id="label"></wicket:container></button>
+        <button wicket:id="autoCreateLabel" type="submit" class="btn btn-primary">autoCreateLabel</button>
```

Comment:
This change is not related to this refactoring. Please create a new pull request for feature changes.

Reply:
I know, teacher. I have removed the commit e17f7fc from the current pull request